### PR TITLE
adding triples gradient timings

### PR DESCRIPTION
**Pull request by hokru**
## Description
Enhances user experience for DF-CCSD(T) gradients.

## Todos
Notable points (developer or user-interest) that this PR has or will accomplish.
- [x] timings for DF-(T) gradient (copied from fnocc module)
- [x] corrects ijk count for gradient (full combination instead of unique pairs)

## Checklist
- [x] relevant tests run (dfccsd-t-grad1)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Comment by dgasmith** [comment]
Can you rebase this with master so that appveyor will pass.

**Comment by hokru** [comment]
Pushed the rebase against today's master.

File: psi4/src/psi4/dfocc/ccsd_triples.cc [hunk 1]
```diff
@@ -1656,7 +1657,9 @@ void DFOCC::ccsd_canonic_triples_grad2() {
     outfile->Printf("\tnthreads: %i \n", nthreads);
 
     // Find number of unique ijk combinations (i>=j>=k)
-    Nijk = naoccA * (naoccA + 1) * (naoccA + 2) / 6;
+    // Nijk = naoccA * (naoccA + 1) * (naoccA + 2) / 6;
+    // Full set of ijk used here
+    Nijk = naoccA * naoccA * naoccA;
```
Comment (dgasmith) [inline psi4/src/psi4/dfocc/ccsd_triples.cc]: Is this quantity used anywhere else? You might consider building a new variable to be safe.

File: psi4/src/psi4/dfocc/ccsd_triples.cc [hunk 2]
```diff
@@ -2098,6 +2105,26 @@ void DFOCC::ccsd_canonic_triples_grad2() {
                 Miabd->contract(false, true, navirA * navirA, navirA, navirA, V, T, 0,
                                 (j * naoccA * navirA * navirA) + (k * navirA * navirA), 1.0, 1.0);
 
+
```
Comment (dgasmith) [inline psi4/src/psi4/dfocc/ccsd_triples.cc]: You might want to consider something like:

```
double step_print = 5.0;
double next_print = step_print;

for (){
    double percent = (double)ind/Nijk
    if (percent >= next_print){
     next_print += step_print;
     print ...
}
```

Should be fewer lines and give a cleaner path to changing the percents later.

File: psi4/src/psi4/dfocc/ccsd_triples.cc (hunk 1 above)
Comment (hokru) [reply psi4/src/psi4/dfocc/ccsd_triples.cc]: Only use is the print out for the user, I assume it was simply (wrong) copy & paste from the energy-only subroutine.

File: psi4/src/psi4/dfocc/ccsd_triples.cc (hunk 2 above)
Comment (hokru) [reply psi4/src/psi4/dfocc/ccsd_triples.cc]: good idea, I added I nicer version.

**Pull request merged**